web/src/js/utils.js

```diff
@@ -105,3 +105,17 @@ fetchApi.put = (url, json, options) => fetchApi(
         ...options
     }
 )
+
+
+export function getDiff(obj1, obj2) {
+    let result = {...obj2};
+    _.forIn(obj1, (value, key) => {
+    	  if(_.isEqual(obj2[key], obj1[key]))
+            result[key] = undefined;
+        else if(typeof Array.isArray(obj2[key]) && Array.isArray(obj2[key]))
+            result[key] = {...obj2[key]};
```

Comment:
yes, i kown. its was just an example. in the getDiff methode i just use the array of the secound obj if they are not equal. --> headers: [...] is an array and so if anything changes in the header, the header will be updated (completly)